s3_poller

```diff
@@ -13,6 +13,7 @@ require 'opscode/version'
 
 class S3Poller
   attr_reader :config
+  attr_writer :projects
```

Comment:
Ack.. I didn't mean to leave this. I started on a slightly different approach and quickly realized it was overkill. I'll push a quick fix.